portal/src/components/Layout/Layout.tsx

```diff
@@ -39,12 +47,51 @@ export const Layout: React.FC<Props> = ({ children, location, pathContext }) =>
         );
     }
 
+    const shouldShowSidebar = !isFrontPage && !(screen.isSmallDevice || screen.isMediumDevice);
+
+    const metaDescription = site.siteMetadata.description;
+    const siteTitle = site.siteMetadata.title;
+    const title = pageContext.frontmatter?.title || "Jøkul Designsystem";
+
     return (
-        <div className="jkl jkl-portal" ref={wrapperRef}>
-            <Helmet>
-                <html lang="no" />
-                <title>{PageTitle}</title>
-            </Helmet>
+        <div className="jkl jkl-portal">
+            <Helmet
+                htmlAttributes={{
+                    lang: "no",
+                }}
+                title={title}
+                titleTemplate={siteTitle ? `%s | ${siteTitle}` : undefined}
```

Comment:
Dette blir vel feil. Nå blir tittelen "Jøkul Designsystem | Jøkul Designsystem" på sider som ikke har noen tittel, siden `siteTitle` _alltid_ er satt. Usikker på hvordan templaten funker i Helmet, men kan dette være en løsning?

```suggestion
                title={pageTitle}
                titleTemplate={pageTitle ? `%s | ${siteTitle}` : siteTitle}
```

Reply:
`siteTitle` er bare `Jøkul`, så for eksempel forsiden får `<title>` `Jøkul Designsystem | Jøkul`. Men åpen for å ta inn endringen din.